charts/telegraf-ds/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: telegraf-ds
-version: 1.0.22
-appVersion: 1.19
+version: 1.0.23
+appVersion: "1.19"
```

Comment:
Could you rebase and increment `version` (with `appVersion` unquoted) please? Thanks.

Reply:
Sure, no problem.